sphinxcontrib/openapi/openapi30.py

```diff
@@ -296,20 +312,20 @@ def _httpresource(endpoint, method, properties, convert, render_examples,
                 query_param_examples.append((param['name'], example))
 
     # print request content
-    if render_request:
+    if True: # render_request:
```

Comment:
Hack, how can I set render_request to True?

Reply:
This change makes tests fail, I can revert it, but I think this is a better solution, we could DRY it up also